openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2200,7 +2200,13 @@ The Credential Format Identifier is `mso_mdoc`. This refers to the Mobile Securi
 
 ### Credential Issuer Metadata {#server-metadata-mso-mdoc}
 
-Cryptographic algorithm names used in the `credential_signing_alg_values_supported` parameter SHOULD be one of those defined in [@!ISO.18013-5].
+Cryptographic algorithm names used in the `credential_signing_alg_values_supported` parameter correspond to the algorithm identifiers used to secure the COSE `IssuerAuth` structure, as defined in [@!ISO.18013-5]. The values SHOULD be chosen from the following:
+
+* The value exactly matches the `alg` value in the IssuerAuth COSE header.
+* The value is a fully specified algorithm, as defined in [I-D.ietf-jose-fully-specified-algorithms], and the combination of the `alg` value and the signing key's curve in the `IssuerAuth` COSE structure matches the combination specified by the fully specified algorithm.
```

Comment:
As I stated in this PR https://github.com/openid/OpenID4VP/pull/553 im concerned about the complexity of this logic, rather than just having a `credential_signing_crv_values_supported` parameter which would mean the wallet nor the issuer would need to maintain mapping logic here as there would always be an explicit match.

Reply:
While I agree that a `_crv` parameter would be slightly less complex, the DCP WG decided not to introduce a `_crv` parameter for OID4VP, and opted for consistency by making the same decision for OID4VCI.

Reply:
I think we should actually require fully specified algorithm for this, otherwise the wallet does not know whether the curve is supported before getting the credential.

Reply:
Discussed in the WG and it makes no sense to require fully specified algorithm only.

Reply:
I added the suggestion raised in the last DCP WG call, so I believe this should be addressed.